qemu/tests/cfg/block_hotplug.cfg

```diff
@@ -14,6 +14,8 @@
     disk_op_cmd = "dd if=%s of=/dev/null bs=1k count=1000 iflag=direct &&"
     disk_op_cmd += " dd if=/dev/zero of=%s bs=1k count=1000 oflag=direct"
     kill_vm_on_error = yes
+    ppc64le,ppc64:
+        virtio_block_pause = 10
```

Comment:
Why not use the same value for both ppc and x86?

Reply:
Hotplug a device, 5s later, x86 can find the device in guest, but ppc can't. So after discussion with yhong, set the time to 10s for ppc.